### ISSUE TITLE
Error updating plugin (node:32011)

### ISSUE THREAD

**Issue by bolex222** [Hyperline/hyperline#194]
get this error at starting hyper 
![warning hyperline](https://user-images.githubusercontent.com/50240788/88223368-a333cc00-cc67-11ea-987a-9907a0e59154.png)

**Comment by thecodelayeriser** [comment]
Have the same issue.

**Comment by samlovescoding** [comment]
Yup. The issue still remains on Linux Mint 19.3 (Cinnamon 4.4.8 / Linux 5.4.0-42-generic).

**Comment by emmanuelnk** [comment]
@bolex222 it's because the dependencies are outdated. Especially babel and webpack. I've made a PR to update them and the errors are taken care of. The plugin works with the latest version of Hyper. 
However, I don't know if this project is still being maintained.

If you want to you can clone my master fork into `~/.hyper_plugins/local/`, run `yarn` and then `yarn run build` and add `'hyperline'` to `localPlugins` array in the `hyper.js` config file.

I'm on Ubuntu 20.04 but I believe it should work for others using latest Hyper.  

My pull request: #195

**Comment by bolex222** [comment]
Thx @emmanuelnk! hope your PR will be accepted

**Comment by abixadamj** [comment]
> I'm on Ubuntu 20.04 but I believe it should work for others using latest Hyper.
> 
> My pull request: #195

@emmanuelnk - Great work ;-)
![Zrzut ekranu z 2020-11-08 19-22-05](https://user-images.githubusercontent.com/5088643/98481516-13586480-21fb-11eb-976b-6a12e0afd425.png)

Works perfectly on Mint 20 LTS.
